src/scripts/core/models/grid.ts

```diff
@@ -65,10 +65,57 @@ export class Grid extends Container {
 
     }
 
+    private matchCheck(inputCoords: number[]): void {
+
+        const originX = inputCoords[0];
+        const originY = inputCoords[1];
+        const origin = this.getToken(originX, originY);
+        let xMatches = []; xMatches.push(origin);
+        let yMatches = []; yMatches.push(origin);
+
+        //check token's right
+        for(let i = originX+1; i <= this.gridSize-1; i++) {
+            if(!this.isMatch(origin, this.getToken(i, originY))) {break;}
```

Comment:
Can these if statements be split onto multiple lines, just for the sake of consistency and readability with our codebase